osmosis-1/osmosis-1.assetlist.json

```diff
@@ -6285,6 +6285,51 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkatom.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkatom.svg"
       }
+    },
+    {
+      "description": "The native staking and collateral coin of Onomy Protocol",
+      "denom_units": [
+        {
+          "denom": "ibc/B9606D347599F0F2FDF82BA3EE339000673B7D274EA50F59494DC51EFCD42163",
+          "exponent": 0,
+          "aliases": [
+            "anom"
+          ]
+        },
+        {
+          "denom": "unom",
```

Comment:
I recommend that if you want to define another denom that you do so instead at the Chain Registry.

Reply:
Removed unom